src/api/TokenIOSync.h

```diff
@@ -132,4 +132,28 @@
                             keyName:(NSString *)keyName
                                 key:(Key *)key;
 
+#pragma mark - Recovery
+
+/**
+ * Begins recovery process for an alias. The verification message will be sent if the alias is valid.
```

Comment:
If you like the idea of changing the comment in TokenIO.h, you're remembering to also change it here.